app/models/api/openid_connect/o_auth_application.rb

```diff
@@ -0,0 +1,66 @@
+module Api
+  module OpenidConnect
+    class OAuthApplication < ActiveRecord::Base
+      has_many :authorizations, dependent: :destroy
```

Comment:
Prefer the new style validations `validates :column, uniqueness: value` over `validates_uniqueness_of`.